stytra/experiments/tracking_experiments.py

```diff
@@ -136,30 +162,99 @@ def wrap_up(self, *args, **kwargs):
 
         self.camera.join()
 
-    def excepthook(self, exctype, value, tb):
+    def _setup_dispatcher(self, recording_event: Event = None) -> DispatchProcess:
```

Comment:
the name of this function is a bit too generic

Reply:
I changed the name to `_setup_frame_dispatcher`. If this is still too generic, please let me know about any suggestions you might have.

Reply:
I changed it to `_setup_frame_dispatcher` now. If you think that's also too generic please let me know any suggestions.